tests/test_utils.py

```diff
@@ -39,6 +39,7 @@
     save,
     send_to_device,
 )
+from accelerate.utils.operations import convert_to_fp32
```

Comment:
We're close. This needs to be moved into where `save` and `send_to_device` is imported from. E.g.

```
from accelerate.utils import (
    ...,
    ...,
    save,
    send_to_device,
    convert_to_fp32,
)
```
(And then run `make style; make quality` as the imports need to be in alphabetical order

Reply:
Sorry, I missed that this is already retrieved in the init file, updated.